processor/intervalprocessor/config.go

```diff
@@ -26,6 +26,9 @@ type Config struct {
 	// SummaryPassThrough is a flag that determines whether summary metrics should be passed through
 	// as they are or aggregated.
 	SummaryPassThrough bool `mapstructure:"summary_pass_through"`
+
+	// Experimental feature to enable aggregation of metrics with time-based partitioning.
+	Partitions int `mapstructure:"partitions"`
```

Comment:
does this really need to be configurable?

technically having too many partitions on very low load might introduce some overhead. If I get to decide, I'd wait for this to be a problem instead of fixing it now.

having another knob brings another thing for users to think about. I believe setting this to 60 for everybody does a good-enough job, at least until someone opens an issue and complains because it didn't

Reply:
Yeah I have the same impression. I'm preferring to not let this configurable, but I was hoping to benchmark this with several different numbers of partitions to see what could be the sweet spot here